source/manual/deploying.html.md

```diff
@@ -17,27 +17,44 @@ If you are responding to a security incident, follow the steps to [deploy fixes
 [reg]: https://gds.blog.gov.uk/2012/11/02/regular-releases-reduce-risk
 [sec]: /manual/deploy-fixes-for-a-security-vulnerability.html
 
-## 1. Test on integration
+## Deploying a branch
+
+It can be useful to test your branch in a non-local environment before it is merged to master. Note that you should only do this on Integration, unless you have a good reason to deploy to a different environment.
```

Comment:
It might also be worth adding something about checking whether there already is a branch deployed and asking the person who deployed it if they are finished.

Reply:
Have now added a preamble before this sentence which should cover it 👍